### PR TITLE
Portfolio Project Accessibility Checks

### PR DESCRIPTION
**Pull request by CaptainBawa**
The basic structure of the page - no issues found.
Moving content - no issues found.
Multimedia - no issues found.
Interaction - no issues found.
Color contrast - no issues found.
Resize - no issues found.
Page tittles - change from my portfolio to my portfolio project homepage.
Text headings - change the hierarchy of my headings.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,45 +16,45 @@
                 <a class="desk-port" href="./index.html">Portfolio</a> 
                 <a class="desk-about" href="./index.html">About</a> 
                 <a class="desk-contact" href="./index.html">Contact</a>      
-                <img class="mailbox" src="./desktop-img/Icon - Mail.svg" alt="message icon">
```
Comment (ShadyShawkat) [inline index.html:19]: - [ ] Kindly make sure that the page has a minimum contrast ratio of at least 4.5:1. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast.

image
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @CaptainBawa, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (CaptainBawa) [reply index.html:19]: @ShadyShawkat 

Please, I tried using the tool you suggested and this is the output I got.
Capture2

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @CaptainBawa and @Kgomotso196 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Requested changes solved :heavy_check_mark: 

## Optional suggestions
- You can consider using the [Wave Tool](https://wave.webaim.org/) to ensure that you have resolved every possible accessibility issue in your project :100: 
- Currently, I see that there are some errors you can improve on so kindly do that even though this is an optional comment :wink: 
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**